### DISCUSSION TITLE
Restricting data to X number of Hospitals

### DISCUSSION THREAD

**Discussion by Data-Wizardry** [synthetichealth/synthea#1327] [Q&A] [answered]
Hello,

When I generate data in Synthea, is there a way to force it to only generate data at a single hospital? Or maybe X number of hospitals?

Thanks!

**Comment by dehall** [comment, marked as answer]
We don't have a direct configuration option for something like that, but two possible approaches that may work depending on your needs:
1. Run synthea for only a single city, for example `./run_synthea Massachusetts Boston`. Individuals will seek care at the closest facility that provides the (high-level) service they need, so if all your patients are in a single city they will only go to the set of closest hospitals. (The hospitals will not necessarily be in that same city, but they will be reasonably close)
2. Reduce the set of hospitals that Synthea knows about - the files are in `src/main/resources/providers/` https://github.com/synthetichealth/synthea/tree/master/src/main/resources/providers
There's a little more info here https://github.com/synthetichealth/synthea/wiki/Provider-Data , but essentially if you delete the rows for all hospitals except for one or just a few, that's where all the patients will necessarily go. You may still want to combine this with running only a single city to make sure the results are realistic. (For example if you run in California, you probably don't want to see patients from Sacramento going to a hospital in San Diego)

**Reply by Data-Wizardry** [reply]
Thanks!
